include/util/pngEncodeDecode.hpp

```diff
@@ -0,0 +1,78 @@
+#pragma once // wtf is this
```

Comment:
This was a best practice I picked up from work, but after consulting cppcoreguidelines, https://github.com/gillwald/geometry/issues/8 we should revert to the standard way until we're using C++120 modules